xmtp_api_grpc/src/grpc_api_helper.rs

```diff
@@ -109,8 +110,7 @@ impl ApiBuilder for ClientBuilder {
         let channel = match self.tls_channel {
             true => create_tls_channel(host, self.limit.unwrap_or(1900)).await?,
             false => {
-                Channel::from_shared(host)?
-                    .rate_limit(self.limit.unwrap_or(1900), Duration::from_secs(60))
+                apply_channel_options(Channel::from_shared(host)?, self.limit.unwrap_or(1900))
```

Comment:
We weren't previously applying all the usual options to non-tls channels, which are what we use in tests.